static/app/views/settings/organizationMembers/inviteMember/orgRoleSelect.tsx

```diff
@@ -29,21 +37,36 @@ class OrganizationRoleSelect extends Component<Props> {
       disabled,
       enforceRetired,
       enforceAllowed,
+      isCurrentUser,
       roleList,
+      idpRoleRestricted,
       roleSelected,
       setSelected,
     } = this.props;
 
+    const panelAlertText = () => {
+      if (isCurrentUser) {
+        return "Your organization-level role is managed through your organization's identity provider.";
+      }
+      return "This member's organization-level role is managed through your organization's identity provider.";
```

Comment:
Might be better to do it further below, in this manner 

```suggestion
      tct("[person] organization-level role is managed through your organization's identity provider.", {person: isCurrentUser ? "Your" : "This member's"});
```